packages/mg-substack-csv/lib/mapper.js

```diff
@@ -1,12 +1,19 @@
+const {formatISO} = require('date-fns');
+
 const mapConfig = (data, {url, readPosts, email}) => {
     const slug = data.post_id.replace(/^(?:\d{1,10}\.)(\S*)/gm, '$1');
 
+    // Get an ISO 8601 date - https://date-fns.org/docs/formatISO
+    const dateNow = formatISO(new Date());
+
     const mappedData = {
         url: `${url}/${slug}`,
         substackId: data.post_id,
         data: {
             slug: slug,
-            published_at: data.post_date,
+            published_at: data.post_date || dateNow,
```

Comment:
Another idea could be to just use the same date. For instance if the `published_at` date exists, we could use this value also for the `updated_at` and `created_at`. wdyt?

Reply:
Maybe I'm missing something, but isn't that what I'm doing here? Or do you mean use `updated_at` and `created_at` if they exist in the source material? 

If it's the latter, WP has created & modified dates, HubSpot has the same, and it looks like the rest only have the one date. This is the route I would've gone down if this was applied to all sources.

Reply:
Oh yeah! I completely misread that! Sorry, all good then 😬

Reply:
😸 I'll move forward with and prepare a full PR adding the same logic to all sources